src/Facade/Responses/GenericSinglePROPFINDCalDAVResponse.php

```
@@ -46,15 +46,15 @@ protected function parse()
         if (isset($this->content['response']['propstat']['prop']) && isset($this->content['response']['propstat']['status'])) {
             // all props found
             $status = $this->content['response']['propstat']['status'];
-            if ($status == AbstractCalDAVResponse::HttpOKStatus) {
+            if ($this->statusMatches($status, AbstractCalDAVResponse::HttpOKStatus)) {
                 $this->found_props = $this->content['response']['propstat']['prop'];
                 $this->not_found_props = null;
```

PHPStan (2.1.46)

[error] Property CalDAVClient\Facade\Responses\GenericSinglePROPFINDCalDAVResponse::$not_found_props (array) does not accept null.


(assign.propertyType)